src/PolykeyAgent.ts

```diff
@@ -99,6 +101,7 @@ class PolykeyAgent {
       connTimeoutTime?: number;
     };
     networkConfig?: NetworkConfig;
+    seedNodes?: NodeMapping;
```

Comment:
Question is what does this "default" to? I believe it should default to `config.defaults.network.mainnet` as well.

Reply:
So the idea is that `PolykeyAgent` defaults to an empty seed nodes.

Reply:
Because it's easier during testing and development.

Reply:
Add a comment about this on `config.ts`.